src/Server/HTTP/authenticateUserByHTTP.cpp

```diff
@@ -79,13 +80,19 @@ bool authenticateUserByHTTP(
     std::string spnego_challenge;
     SSLCertificateSubjects certificate_subjects;
 
+    if (config_credentials)
+    {
+        checkUserNameNotEmpty(config_credentials->getUserName(), "config authentication");
+    }
```

Comment:
Just for consistency this check should be done before you copy `config_credentials` into `current_credentials`.

Reply:
But copying is done way later, at L206